provider.go

```diff
@@ -7,6 +7,9 @@ type Provider interface {
 	// Name returns the provider name.
 	Name() string
 
+	// Project returns project name for GCP, profile name for AWS and subscription name for Azure
```

Comment:
Missing dot at the end. Also please fill the paragraph to ~80 cols like the ones above.